### DISCUSSION TITLE
X360 Compability

### DISCUSSION THREAD

**Discussion by Szoszenka** [Berny23/LD-ToyPad-Emulator#36] [Q&A]
Has anyone checked if the emulator works with the Xbox 360?

**Comment by cort1237** [comment]
I'm not sure any One has actually tried. But it's very unlikely to work. Xbox require additional USB encryption in so their peripherals that isn't present here. This encryption isn't well analyzed so we have no way of recreating it.

**Reply by Szoszenka** [reply]
Well that aint good news, but thanks for the answer

**Reply by wertercatt** [reply]
The Xbox 360 Toypad also has a different idVendor and idProduct from the Toypad that this repository is emulating.
See attached Get-PnpDeviceProperty output: [LD-Toypad-Xbox-360.csv](https://github.com/Berny23/LD-ToyPad-Emulator/files/8036394/LD-Toypad-Xbox-360.csv)
